docs/open-telemetry.asciidoc

```diff
@@ -15,480 +16,72 @@
-<3> The https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter[logging exporter] is helpful for troubleshooting and supports various logging levels: `debug`, `info`, `warn`, and `error`.
-<4> Elastic {observability} endpoint configuration.
-APM Server supports a ProtoBuf payload via both the OTLP protocol over gRPC transport {ot-grpc}[(OTLP/gRPC)]
-and the OTLP protocol over HTTP transport {ot-http}[(OTLP/HTTP)].
-To learn more about these exporters, see the OpenTelemetry Collector documentation:
-https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter[OTLP/HTTP Exporter] or
-https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlpexporter[OTLP/gRPC exporter].
-<5> Hostname and port of the APM Server endpoint. For example, `elastic-apm-server:8200`.
-<6> Credential for Elastic APM <<secret-token,secret token authorization>> (`Authorization: "Bearer a_secret_token"`) or <<api-key,API key authorization>> (`Authorization: "ApiKey an_api_key"`).
-<7> Environment-specific configuration parameters can be conveniently passed in as environment variables documented https://opentelemetry.io/docs/collector/configuration/#configuration-environment-variables[here] (e.g. `ELASTIC_APM_SERVER_ENDPOINT` and `ELASTIC_APM_SECRET_TOKEN`).
-<8> To send OpenTelemetry logs to {stack} version 8.0+, declare a `logs` pipeline.
-
-You're now ready to export traces and metrics from your services and applications.
-
 [float]
-[[open-telemetry-collect-metrics]]
-==== Collect metrics
-
-IMPORTANT: When collecting metrics, please note that the https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-api/latest/io/opentelemetry/api/metrics/DoubleValueRecorder.html[`DoubleValueRecorder`]
-and https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-api/latest/io/opentelemetry/api/metrics/LongValueObserver.html[`LongValueRecorder`] metrics are not yet supported.
-
-Here's an example of how to capture business metrics from a Java application.
-
-[source,java]
-----
-// initialize metric
-Meter meter = GlobalMetricsProvider.getMeter("my-frontend");
-DoubleCounter orderValueCounter = meter.doubleCounterBuilder("order_value").build();
+== OpenTelemetry and the {stack}
 
-public void createOrder(HttpServletRequest request) {
+[subs=attributes+]
+include::./diagrams/apm-otel-architecture.asciidoc[Architecture of Elastic APM with OpenTelemetry]
 
-   // create order in the database
-   ...
-   // increment business metrics for monitoring
-   orderValueCounter.add(orderPrice);
-}
-----
+Elastic integrates with OpenTelemetry, allowing you to reuse your existing instrumentation
+to easily send observability data to the {stack}.
+There are four ways to integrate OpenTelemetry with the {stack}:
```

Comment:
@estolfo Do you know who is working on the blog post with pros/cons of each approach?

Reply:
I think @AlexanderWert and @gregkalapos were involved with the blog post. We might have one specifically on the OTel bridge before we have a pros/cons one because this is the tracking issue and it only mentions the bridge: https://github.com/elastic/observability-technical-assets/issues/34

Reply:
👍 I'll remove that from this PR so we're not blocked.